draft-ietf-dnsop-svcb-httpssvc.md

```diff
@@ -130,51 +130,77 @@ Perhaps this could be included as a svc="baz" parameter
 for protocols that are not the default for the RR type?
 Current inclination is to not do so.
 
-## Introductory Example
+## Example: Protocol enhancements
 
-As an introductory example for an HTTPS origin resource, a set of
-example HTTPSSVC and associated A+AAAA records might be:
+Consider a simple zone of the form
 
-    www.example.com.  7200  IN CNAME    svc.example.net.
-    ; AliasForm
-    example.com.      7200  IN HTTPSSVC 0 svc.example.net.
-    ; ServiceForm
-    svc.example.net.  7200  IN HTTPSSVC 2 svc3.example.net. ( alpn=h3
-                                        port=8003 esniconfig="..." )
-    svc.example.net.  7200  IN HTTPSSVC 3 svc2.example.net. ( alpn=h2
-                                        port=8002 esniconfig="..." )
-    svc2.example.net. 300   IN A        192.0.2.2
-    svc2.example.net. 300   IN AAAA     2001:db8::2
-    svc3.example.net. 300   IN A        192.0.2.3
-    svc3.example.net. 300   IN AAAA     2001:db8::3
-    ; Compatibility records for non-HTTPSSVC-aware clients
-    example.com.      300   IN A        192.0.2.1
-    example.com.      300   IN AAAA     2001:db8::1
-    svc.example.net.  300   IN A        192.0.2.1
-    svc.example.net.  300   IN AAAA     2001:db8::1
-
-In the preceding example, both of the "example.com" and
-"www.example.com" origin names are aliased to use alternative service
-endpoints offered as "svc.example.net" (with "www.example.com"
-continuing to use a CNAME alias).  HTTP/2 is available on a cluster of
-machines located at svc2.example.net with TCP port 8002 and HTTP/3 is
-available on a cluster of machines located at svc3.example.net with
-UDP port 8003.  The client can use the specified ESNI keys to encrypt
-the SNI values of "example.com" and "www.example.com" in the handshake
-with these alternative service endpoints.  When connecting, clients will
-continue to treat the authoritative origins as "https://example.com"
-and "https://www.example.com", respectively.
-
-For services other than HTTPS (as well as for HTTPS origins
-with non-default ports), the SVCB RR and an {{?Attrleaf}} label will be used.
-For example, to reach an example resource of
+    simple.example. 300 IN A    192.0.2.1
+                           AAAA 2001:db8::1
+
+The domain owner could publish new records like
```

Comment:
replace "new" with "added" or "additional".  ("new" might imply that they replace the A and AAAA records.

Reply:
Done